src/osp/Active/SysAreaAssociate.h

```diff
@@ -36,9 +36,7 @@ namespace osp::active
 
 
 struct ACompAreaLink
-{
-    using MapSatToEnt_t = std::unordered_map<universe::Satellite, ActiveEnt>;
-    
+{    
```

Comment:
Std::move satalite on the AConpAreaLubj constructor?

Reply:
Is a constructor actually needed for thia type? What about aggregate initialization?

Reply:
Satellite is just an integer, and doesn't need move. `m_enter`, `m_leave`, and `m_move` are always going to be default initialized; the constructor is just for convenience.